modules/my-module/android/build.gradle

```diff
@@ -32,6 +33,7 @@ buildscript {
 
   dependencies {
     classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${getKotlinVersion()}")
+    classpath "org.jetbrains.kotlin:kotlin-serialization:${getKotlinVersion()}"
```

Comment:
Not your fault. So why is this not in the ducks ducks about this?

Reply:
it does mention it if you search https://github.com/Kotlin/kotlinx.serialization?tab=readme-ov-file for "classpath" you see it.

I just don't know kotlin that well x.x